examples/remote-ui/app/remote/remote.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsxRuntime automatic */
+/** @jsxImportSource react */
+import {createEndpoint, fromInsideIframe, retain} from '@remote-ui/rpc';
```

Comment:
If you do merge this with the kitchen sink example, IMO it's fine to use the `@quilted/threads` library here instead of Remote UI’s RPC library.

Reply:
Right, but nobody uses `@quilted/threads` with remote-ui. The idea was to have the example as close to what's actually out there today as possible.